code/modules/mob/living/simple_animal/friendly/lizard.dm

```diff
@@ -24,13 +24,66 @@
 	can_collar = TRUE
 	mob_biotypes = MOB_ORGANIC | MOB_BEAST | MOB_REPTILE
 	gold_core_spawnable = FRIENDLY_SPAWN
+	var/eats_bugs = TRUE
+	var/eating_sound = 'sound/weapons/bite.ogg'
+	/// Lizards start with a tail
+	var/still_has_tail = TRUE 
+
+/mob/living/simple_animal/lizard/handle_automated_action()
+	if(!stat && !buckled)
+		if(prob(1))
+			custom_emote(EMOTE_VISIBLE, pick("sticks out its tongue", "wags its tail.", "lies down."))
+
+	if(eats_bugs && isturf(loc) && !incapacitated())
+		for(var/mob/living/M in view(1, src))
+			if(!M.stat && Adjacent(M) && HAS_TRAIT(M, TRAIT_EDIBLE_BUG))
+				custom_emote(EMOTE_VISIBLE, "eats \the [M]!")
+				playsound(loc, eating_sound, 20, 1)
+				M.death()
+				stop_automated_movement = FALSE
```

Comment:
I set this to false because I'm an idiot and didn't remove all of the previous implementation of this I scrapped, thanks for pointing this out I have no idea how I missed this